### DISCUSSION TITLE
Hackear GitHub issues para organização pessoal, organização de empresas, transparências e interações ricas.

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#448] [Discussão]
Na issue #411 rolou uma discussão sobre profissionais novos em uma empresa. Depois rolou uma conversa #446 sobre como organizar times de front-end, e isso me leva a escrever essa issue.

Eu sei que a pergunta é ampla, e acredito muito nas interações que fazemos no GitHub, é um local extremamente transparente, onde geralmente pensamos para clicar em enviar e por isso deixamos as discussões muito ricas de informação.

Recentemente vi pessoas usando ele como agenda pessoal, colocando um todo list de tudo o que precisa fazer, e ir fechando as issues conforme for finalizando, até mesmo usando milestones.

O GitLab que é parecido com o GitHub usa muito a própria ferramenta para gerenciar tickets, e é uma empresa 100% remota https://about.gitlab.com/handbook/#communication usando ferramentas de live para conversar.

Dentro da empresa eu sempre digo que este forum é um exemplo de comunicação, pois mencionar assuntos por issues, e poder mencionar até via código é algo bem interessante, sem contar com imagens, reações, milestone, projects etc... e tudo pode ser encontrado depois pela busca busca do GitHub a missão de deixar a gente encontrar assuntos, e tópicos.

Como vocês veem isso, e como você hackearia a interface do GitHub para fazer as coisas mais malucas no mundo?

## Dúvida...

No seu trabalho anterior ou atual, quais ferramentas usavam para se organizar e como você acha que com o GitHub você poderia organizar melhor o seu trabalho e entregas.

😄

**Comment by felquis** [comment]
Relacionado #61

**Comment by bernardodiasc** [comment]
Pergunta interessante. Faço algo aqui que considero útil, não é bem hackear, mas é útil. Explicando simplificadamente, faço clone do wiki dentro da pasta do projeto e incluo "wiki" no .gitignore, depois rodo um script (feito em casa mesmo) node.js pra copiar os arquivos MD do code base  para a pasta wiki e baseado num json estatico de config monto o `_sidebar.md`. Dessa forma a documentaçao que esta no code base fica organizada no wiki com bem pouco esforço.

**Comment by mstuttgart** [comment]
Eu utilizo o github pra organizar meus estudos: https://github.com/mstuttgart/estudos/projects/2?fullscreen=true
O lance de você ter labels, milestones e poder usar markdown ajuda muito.

**Comment by MrS4w** [comment]
Faço parecido com o @mstuttgart 
https://github.com/MrS4w/courses
